lnst/RecipeCommon/Perf/Evaluators/NonzeroFlowEvaluator.py

```diff
@@ -43,6 +43,12 @@ def evaluate_results(
                             )
                         )
                     elif metric.average > 0:
+                        for interval in metric:
+                            if interval.value == 0:
+                                result = False
+                                result_text.append(
+                                    "{} reported zero throughput".format(metric_name)
+                                )
```

Comment:
With this implementation it may happen that due to average not being zero, the report may contain both the "zero" and "non-zero". Maybe add an "else" statement for the for loop.

Reply:
Oh right, I was testing it only with unit tests and I did not see any output even in aforementioned case.

I will fix this.